### PR TITLE
WIP | Initial project setup

### PR DESCRIPTION
**Pull request by bhumitaylor14**
## Description
WIP commit  - project setup through npm library standards

## Motivation and Context
Develop Etherspot UI library for sending native token.

## How Has This Been Tested?
On local machine

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ tick] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (IAmKio) [inline .gitignore:1]: Can you please find a  better .gitignore templates for projects like this please?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<!-- CONTRIBUTING -->
+## Contributing
```
Comment (IAmKio) [inline CONTRIBUTING.md:2]: Please copy the content from other Etherspot projects here

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Changelog
+
```
Comment (IAmKio) [inline CHANGELOG.md:2]: You've included a package.json update in this PR, please add the change log here

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Reporting security issues
+
```
Comment (IAmKio) [inline SECURITY.md:2]: Please ask DevOps what should go here.

File: __tests__/index.test.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+// index.test.js
+import React from 'react';
+import renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import TestModule from '../src/index';
+
+test('TestModule renders correctly', () => {
+  const tree = renderer.create(<TestModule />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (IAmKio) [inline __tests__/index.test.js:10]: Remember newline at the end of the file, do you have editorconfig set up?

File: jest.config.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+      '^.+\\.(js|jsx)?$': 'babel-jest',
+    },
+  }; */
+  module.exports = {
+    testEnvironment: 'jsdom',
+    moduleFileExtensions: ['js', 'jsx', 'json', 'node'],
+    setupFilesAfterEnv: ['@testing-library/jest-dom/extend-expect'],
+   
+    // ...other configurations
+  };
```
Comment (IAmKio) [inline jest.config.js:14]: Remember new line here: https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

File: package.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "etherspot-ui",
+  "version": "1.0.0",
```
Comment (IAmKio) [inline package.json:3]: Should this really be `1.0.0`?

File: package.json [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  "scripts": {
+    "test": "jest __tests__ --silent"
+  },
+  "keywords": [],
+  "author": "Etherspot Team",
+  "license": "MIT",
+  "dependencies": {
+    "@etherspot/transaction-kit": "^0.6.11",
+    "ethers": "^5.4.0",
+    "lodash": "^4.17.21"
```
Comment (IAmKio) [inline package.json:15]: Please lock these dependencies to prevent code from creeping in

File: src/index.js [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+ * SOFTWARE.
+ */
+
+import React from 'react';
+
+const TestModule = () => {
+  return <div>Hello Etherspot UI</div>;
+};
+
+export default TestModule;
```
Comment (IAmKio) [inline src/index.js:30]: New line

**Review by IAmKio — CHANGES_REQUESTED** [review]

**Comment by bhumitaylor14** [comment]
@IAmKio , I have added code for your suggested changes on latest PR

**Pull request closed without merging**